source/frontend/src/index.tsx

```diff
@@ -41,19 +44,44 @@ async function prepare() {
 
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 //@ts-ignore
-const keycloak: KeycloakInstance = new Keycloak('/keycloak.json');
+const keycloak: Keycloak = new Keycloak('/keycloak.json');
 const Index = () => {
   return (
     <TenantProvider>
       <ModalContextProvider>
-        <TenantConsumer>{({ tenant }) => <InnerComponent tenant={tenant} />}</TenantConsumer>
+        <Router>
+          <TenantConsumer>{({ tenant }) => <InnerComponent tenant={tenant} />}</TenantConsumer>
+        </Router>
       </ModalContextProvider>
     </TenantProvider>
   );
 };
 
 const InnerComponent = ({ tenant }: { tenant: ITenantConfig2 }) => {
   const refresh = useRefreshSiteminder();
+
+  // get telemetry configuration from tenant json
+  if (tenant?.telemetry?.enabled) {
+    const config: TelemetryConfig = {
+      name: tenant?.telemetry?.serviceName ?? 'frontend',
+      appVersion: import.meta.env.VITE_PACKAGE_VERSION ?? '',
+      environment: tenant?.telemetry?.environment || 'local',
+      otlpEndpoint: tenant?.telemetry?.endpoint || '',
+      debug: tenant?.telemetry?.debug ?? false,
+      exportInterval: tenant?.telemetry?.exportInterval ?? 30_000,
```

Comment:
30_000?

Reply:
this is the same as 30000 ms (or 30 seconds). If you are asking about the numeric separator - they are supported by JS
https://backbencher-dev.medium.com/numeric-separators-in-javascript-29b154a0297a

```js
console.log(1000_000_000 === 1000000000); // true
```